packages/actor-http-native/lib/Requester.ts

```diff
@@ -57,6 +57,20 @@ export class Requester {
     return requestProxy;
   }
 
+  // Wrap headers into an header object type
+  public convertFetchHeadersToHash(response: any): any {
+    const responseHeaders: Headers = new Headers();
+    if (response.input && response.input.headers) {
```

Comment:
`response.input` is not part of the public `IncomingMessage` interface in the Node request API AFAIK. So this should not be needed.

Reply:
Also, I guess the test mock may be wrong: https://github.com/comunica/comunica/blob/379553bfcf265a46cd91c8a69a4dccd2f0725f0d/packages/actor-http-native/test/__mocks__/follow-redirects.js#L15